web/src/engine/predictive-text/worker-thread/src/tests/mocha/cases/worker-model-compositor.js

```diff
@@ -177,7 +222,9 @@ describe('ModelCompositor', function() {
           insert: "l",
           deleteLeft: 0
         }, {
-          left: "the apl", startOfBuffer: true, endOfBuffer: true
+          // Context + insert needs to match a word in the model; corrections are off with a single-transform
+          // input.
+          left: "the app", startOfBuffer: true, endOfBuffer: true
```

Comment:
The changes affected one pre-existing unit test, as its hard-coded input style matched the "do not correct" pattern.